Assets/MRTK/Core/Definitions/BoundarySystem/MixedRealityBoundaryVisualizationProfile.cs

```diff
@@ -12,7 +12,7 @@ namespace Microsoft.MixedReality.Toolkit.Boundary
     /// </summary>
     [CreateAssetMenu(menuName = "Mixed Reality Toolkit/Profiles/Mixed Reality Boundary Visualization Profile", fileName = "MixedRealityBoundaryVisualizationProfile", order = (int)CreateProfileMenuItemIndices.BoundaryVisualization)]
     [MixedRealityServiceProfile(typeof(IMixedRealityBoundarySystem))]
-    [HelpURL("https://microsoft.github.io/MixedRealityToolkit-Unity/Documentation/Boundary/BoundarySystemGettingStarted.html")]
```

Comment:
Do we have any plans for forwarding along our github.io docs to docs.microsoft.com? Changing these URLs in the repo for past releases won't really change anything, since the code is already released to UPM etc with the old URLs

Reply:
Really, we probably should have been embedding a specific release version in our release docs :( and then we'd just keep all the old versions alive for compatibility